tests/lib/rules/no-implied-eval.js

```diff
@@ -168,90 +200,162 @@ ruleTester.run("no-implied-eval", rule, {
 		},
 		{
 			code: "global.setTimeout(function() { x = 1; }, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.setInterval(function() { x = 1; }, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.execScript(function() { x = 1; }, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.setTimeout(foo, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.setInterval(foo, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.execScript(foo, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "globalThis.setTimeout(foo, 100);",
-			languageOptions: { ecmaVersion: 2020 },
+			languageOptions: { ecmaVersion: 2020, globals: globals.browser },
```

Comment:
```suggestion
			languageOptions: { ecmaVersion: 2020 },
```

`globalThis` is enabled as a global by `ecmaVersion: 2020`.